documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -40,7 +33,11 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 +
 {project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. The details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
 
-. VMware source providers only (All optional):
+ifdef::vmware[]
++
+[IMPORTANT]
+====
+Check the following VMware-specific items. All are optional, but need special attention:
```

Comment:
Check the following VMware items, all of which are optional:

(Do we need the second sentence, especially "special attention"?  This could be interpreted a variety of ways.)